dnf-behave-tests/features/plugins-core/builddep.feature

```diff
@@ -0,0 +1,31 @@
+@wip
+Feature: dnf builddep command
+
+
+Background: Enable builddep plugin
+  Given I enable plugin "builddep"
+    And I use repository "dnf-ci-fedora"
+
+
+Scenario: Dnf builddep can use spec file from a remote location
```

Comment:
Also the test suite still might be run against rhel-8.2, we probably should tag the scenario with `@not.with_os=rhel__eq__8`